demo/runtime/src/runtime/democracy.rs

```diff
@@ -23,12 +23,21 @@ use runtime_support::storage;
 use demo_primitives::{AccountId, Hash, BlockNumber};
 use dispatch::PrivCall as Proposal;
 use runtime::{staking, system, session};
-use runtime::system::PrivPass;
 use runtime::staking::{PublicPass, Balance};
 
+/// A token for privileged dispatch. Can only be created in this module.
+pub struct PrivPass;
```

Comment:
This struct can be created anywhere. It would need a private field to prevent creation in other modules.